command/scp.go

```diff
@@ -72,7 +72,9 @@ func createEnvironmentSCPSubCommands(cfg *config.Config, scpOpts scp.Options) ([
 
 // create an array of environment group sub-commands available to `scp env`
 func createEnvironmentGroupSCPSubCommands(env config.Environment, cfg *config.Config, scpOpts scp.Options) ([]*cobra.Command, error) {
-	groups, err := ansible.GetGroupsForEnvironment(cfg.DPSetupPath, env.Name)
+	path := cfg.GetPath(env)
+
+	groups, err := ansible.GetGroupsForEnvironment(path, env.Name)
```

Comment:
Will this handle empty path?

Reply:
As it defaults to DPSetupPath in the config its fine. When this isn't set in dp-cli-config.yml the application will warn and not create the group commands.